app/src/org/commcare/activities/InstallFromListActivity.java

```diff
@@ -422,4 +414,22 @@ public boolean onOptionsItemSelected(MenuItem item) {
         return super.onOptionsItemSelected(item);
     }
 
+    private void loadPreviouslyRetrievedAvailableApps() {
+        for (AppAvailableToInstall availableApp : storage()) {
```

Comment:
Hm interesting thought but apparently not, because when you open this activity for the first time, storage() will always have no records, and that works just fine